core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/coprocessor/transactional/TrxRegionEndpoint.java.tmpl

```diff
@@ -359,7 +359,7 @@ CoprocessorService, Coprocessor {
   private static final int DEFAULT_SLEEP = 60 * 1000;
   private static final int DEFAULT_MEMORY_THRESHOLD = 100; // 100% memory used
```

Comment:
In checkMemoryUsage(), it will check the current memory usage with this DEFAULT_MEMORY_THRESHOLD, if I change the DEFAULT_MEMORY_THRESHOLD, that logic will break. Here the change only change the startup value. The name of DEFAULT_MEMORY_THRESHOLD is confusing, it is not only setting a default value, but also setting a low watermark , so I cannot simply change that value.